ga4gh/datamodel/datasets.py

```diff
@@ -190,3 +237,70 @@ def _setMetadata(self):
                 except KeyError as err:
                     raise exceptions.MissingDatasetMetadataException(
                         metadataFileName, str(err))
+
+
+class AbstractOntology:
+    """
+    The base class of storing an ontology
+    """
+
+    def __init__(self):
+        self.nameById = dict()
```

Comment:
These should be private variables. We should follow the usual conventions for these maps too, _[something]IdMap and _[something]NameMap. I guess [something] can be empty here?